### ISSUE TITLE
cyclops bug

### ISSUE THREAD

**Issue by FoxPurple** [adam9899/Issues#565]
So I died from cyclops when it picked me up while fighting it, respawned back, got my stuff back but my armor was gone except my boots. Nowhere to be found, I wouldn't find it weird except same thing happened to my friend except she had only helmet left. We both had Dark steel gear.

**Comment by Avalonzo** [comment]
When you say dark steel gear do the enderio armor set or tinkers armor made from dark steel? if enderio set did any of the missing pieces have upgrades in them? if yes and they were in your grave then the armor was straight up destroyed this happened to me wearing enderio end steel armor only leaving my boots and all the upgrades of the missing pieces.

**Comment by FoxPurple** [comment]
Yup it was enderio ones, I didn't have any upgrades except my boots but my gf had all upgraded but she lost everything except helmet, and it wasn't soulbound

**Comment by Krono5** [comment]
Confirmed, same for me including Dark Armour and Ice and Fire Armour.
